src/eduid_userdb/proofing/proofingdb.py

```diff
@@ -250,6 +286,55 @@ def remove_state(self, state):
         self.remove_document({'eduPersonPrincipalName': state.eppn,
                         'verification.email': state.verification.email})
 
+    def remove_state_phone(self, state):
+        """
+        :param state: ProofingStateClass object
+
+        :type state: ProofingStateClass
+        """
+        self.remove_document({'eduPersonPrincipalName': state.eppn,
+                              'verification.number': state.verification.number})
+
+
+class PhoneProofingStateDB(EmailProofingStateDB):
```

Comment:
Do PhoneProofingStateDB need to inherit from EmailProofingStateDB? Isn't ProofingStateDB enough?